src/api/types.ts

```diff
@@ -169,3 +183,21 @@ export interface DeleteStatusResponse {
   status: ResponseStatus;
   message: string;
 }
+export interface SearchRequest {
+  query: string;
+  limit?: number;
+}
+export interface SearchResponse {
+  status: ResponseStatus;
```

Comment:
You forgot to add a `message` field here.

Reply:
thank you! fixed here https://github.com/refstudio/refstudio/pull/313/commits/1bb83ccd797ffd09f0b74dead9c4c17a8f9bbd74